recipe/linter_hints/hints.toml

```diff
@@ -41,3 +41,36 @@ boost = """\
     The `boost` output has been superseded by `libboost-python-devel` (as of \
     1.82), which now comes with a run-export (on `libboost-python`) as well."""
 grpc-cpp = "The `grpc-cpp` output has been superseded by `libgrpc`."
+libgcc-ng = """\
+    This output has been superseded by `libgcc`. Note however, that except in \
+    truly exceptional cases, you should not have to add this manually; you can \
+    rely  on the fact that `{{ compiler("c") }}` and `{{ compiler("cxx") }}` \
+    will always create the correct run-export for this. If you need to _ignore_ \
+    the run-export for whatever reason, the best way to do it is:
+    ```yaml
+    build:
+      ignore_run_exports_from:
+        - {{ compiler("c") }}
+    ```"""
```

Comment:
It's possible to strip the leading whitespace here without dedenting the code; however, I've left it as-is, because having extra indentation should actually help this get rendered correctly in GH comments when put in a list.